### DISCUSSION TITLE
Possible next priorities for Florian

### DISCUSSION THREAD

**Discussion by keithschacht** [AllYourBot/hostedgpt#151] [General]
Hi @florianfelsing, great work helping to polish up the design. You have a really good eye! I created an Issue to capture those additional items you noticed (Issue #149) but I bumped it to v1.0 milestone since I think we have some things that are worth prioritizing ahead of it. So let's defer those.

I'd love to get more help on this project, if you're available! What kind of tasks do you most enjoy helping with?

If you would like to help with some additional design/tailwind tasks, this is a good one:
* https://github.com/the-dot-bot/hostedgpt/issues/119
* https://github.com/the-dot-bot/hostedgpt/issues/134

Are you good with javascript and are you familiar with the rails stimulus pattern? These are a couple that are good JS starters:
* https://github.com/the-dot-bot/hostedgpt/issues/109

Or if you're good with rails, this is a kind of medium difficulty rails task that follows the new turbo pattern:
* https://github.com/the-dot-bot/hostedgpt/issues/57

**Comment by florianfelsing** [comment]
Thanks @krschacht - and I really appreciated your thorough review of my PR, as well.

Would love to take care of these two for now:

- https://github.com/the-dot-bot/hostedgpt/issues/109
- https://github.com/the-dot-bot/hostedgpt/issues/134

**Reply by robacarp** [reply]
Welcome @florianfelsing!

**Reply by florianfelsing** [reply]
Nice to meet you @robacarp!

**Comment by keithschacht** [comment]
That's great, I tried assigning those Issues to you so no one else starts on them, but it looks like I'm not able to. I bet you have to comment on them and then github will let me.

**Comment by keithschacht** [comment]
Oh, it looks like @Kaleshe has taken on #109 ! So Florian, skip that one. If you'd like to do some stimulus work, these are similar in scope:

#135 and #136 are similar in scope
